Src/DeUrgenta.Admin.Api/Validators/PayloadValidators/BlogPostPayloadValidator.cs

```diff
@@ -0,0 +1,15 @@
+﻿using DeUrgenta.Admin.Api.Models;
+using FluentValidation;
+
+namespace DeUrgenta.Admin.Api.Validators.PayloadValidators
```

Comment:
this comment will apply to all validators added:
* please format files 
* rename class to *RequestValidator
* rename folder to RequestValidators
* cleanup request models and remove asp.net validation